.github/workflows/r.yml

```diff
@@ -210,13 +210,12 @@ jobs:
       - uses: r-lib/actions/setup-r@master
         with:
           rtools-version: 40
-          r-version: "4.0"
+          r-version: "4.1"
           Ncpus: 2
       - uses: r-lib/actions/setup-r@master
-        if: ${{ matrix.rtools == 35 }}
         with:
-          rtools-version: 35
-          r-version: "3.6"
+          rtools-version: 40
+          r-version: "4.0"
           Ncpus: 2
```

Comment:
I think that you'll want to get rid of the second `uses: r-lib/actions/setup-r@master` block here